packages/react/toast/src/Toast.tsx

```diff
@@ -157,14 +157,16 @@ const ToastViewport = React.forwardRef<ToastViewportElement, ToastViewportProps>
       if (wrapper && viewport) {
         const handlePause = () => {
           const pauseEvent = new CustomEvent(VIEWPORT_PAUSE);
-          dispatchDiscreteCustomEvent(viewport, pauseEvent);
+          viewport.dispatchEvent(pauseEvent);
           context.isClosePausedRef.current = true;
         };
+
         const handleResume = () => {
           const resumeEvent = new CustomEvent(VIEWPORT_RESUME);
-          dispatchDiscreteCustomEvent(viewport, resumeEvent);
+          viewport.dispatchEvent(resumeEvent);
```

Comment:
Just so I understand, the pointerenter/pointerleave ones are excluded too because they are not considered discrete by React?

btw, it might be nice to add a link to https://github.com/facebook/react/blob/a8a4742f1c54493df00da648a3f9d26e3db9c8b5/packages/react-dom/src/events/ReactDOMEventListener.js#L294-L350 directly in that big comment in the primitive file.

Reply:
Exactly

> it might be nice to add a link

Good shout